docs/SECURITY.rst

```diff
@@ -1,4 +1,4 @@
-Filesytem Permissions
+Filesystem Permissions
 =====================
```

Comment:
Thanks a lot!
You should also add a "=" in line 2 to match the length of the sentence in line 1 :)

Reply:
two commits for two bytes.. overkill :(

Reply:
No problem, we'll squash merge them into one. 😆